core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -170,6 +174,10 @@
     private String serviceLoggerPrefix = DEFAULT_SERVICE_LOGGER_PREFIX;
     private AccessLogWriter accessLogWriter = AccessLogWriter.disabled();
     private boolean shutdownAccessLogWriterOnStop = true;
+    private List<AsciiString> clientAddressHeaders =
+            ImmutableList.of(HttpHeaderNames.FORWARDED, HttpHeaderNames.X_FORWARDED_FOR);
+    private Predicate<InetSocketAddress> clientAddressTrustedProxyFilter = remoteAddress -> false;
```

Comment:
I wonder if we should accept `InetAddress` instead of `InetSocketAddress`. Thoughts?